pipeline/filters/kubernetes.md

```diff
@@ -381,6 +381,26 @@ If you are not seeing metadata added to your kubernetes logs and see the followi
 
 When Fluent Bit is deployed as a DaemonSet it generally runs with specific roles that allow the application to talk to the Kubernetes API server. If you are deployed in a more restricted environment check that all the Kubernetes roles are set correctly.
 
+You can test this by running the following command (replace `fluentbit-system` with the namespace where your fluentbit is installed)
+```text
+kubectl auth can-i list pods --as=system:serviceaccount:fluentbit-system:fluentbit
+```
+If set roles are configured correctly, it should simply respond with `yes`. 
+
+For instance, using Azure AKS, running the above command may respond with: 
+```text
+no - Azure does not have opinion for this user.
```

Comment:
Is this the real message? It seems to be missing the word "an" before "opinion" so just want to make sure we get the exact (current) message.

Reply:
That is actually the real text - copypasted verbatim from my terminal. Don't hate the player, hate the.. cloud provider? ;)

Reply:
Always! 😄